### PR TITLE
feat: new icon - linked-squares

### PR DESCRIPTION
**Pull request by paja1t**
#### Changes proposed in this pull request:

Adds a new `app-pairing` icon

#### Icons

20px: app-pairing

16px: app-pairing

### CODE REVIEW

File: resources/icons/16px/app-pairing.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M2 3V9C2.55228 9 3 9.44772 3 10C3 10.5523 2.55228 11 2 11H1.25C0.559644 11 0 10.4404 0 9.75L0 2.25C0 1.55965 0.559643 1 1.25 1H10.75C11.4404 1 12 1.55965 12 2.25V9.75C12 10.4404 11.4404 11 10.75 11H8C7.44772 11 7 10.5523 7 10C7 9.44772 7.44772 9 8 9H10V3H2Z" fill="#111418"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M14 13V7C13.4477 7 13 6.55228 13 6C13 5.44771 13.4477 5 14 5H14.75C15.4404 5 16 5.55964 16 6.25V13.75C16 14.4404 15.4404 15 14.75 15H5.25C4.55964 15 4 14.4404 4 13.75V6.25C4 5.55965 4.55964 5 5.25 5H8C8.55228 5 9 5.44771 9 6C9 6.55228 8.55228 7 8 7H6V13H14Z" fill="#111418"/>
+</svg>
```
Comment (ggdouglas) [inline resources/icons/16px/app-pairing.svg]: Can we remove the `width`, `height`, `fill`, `fill-rule`, and `clip-rule`, properties from svg and path? They aren't necessary in this instance:
```svg
<svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
<path d="M2 3V9C2.55228 9 3 9.44772 3 10C3 10.5523 2.55228 11 2 11H1.25C0.559644 11 0 10.4404 0 9.75L0 2.25C0 1.55965 0.559643 1 1.25 1H10.75C11.4404 1 12 1.55965 12 2.25V9.75C12 10.4404 11.4404 11 10.75 11H8C7.44772 11 7 10.5523 7 10C7 9.44772 7.44772 9 8 9H10V3H2Z"/>
<path d="M14 13V7C13.4477 7 13 6.55228 13 6C13 5.44771 13.4477 5 14 5H14.75C15.4404 5 16 5.55964 16 6.25V13.75C16 14.4404 15.4404 15 14.75 15H5.25C4.55964 15 4 14.4404 4 13.75V6.25C4 5.55965 4.55964 5 5.25 5H8C8.55228 5 9 5.44771 9 6C9 6.55228 8.55228 7 8 7H6V13H14Z"/>
</svg>
```
app-pairing

File: resources/icons/20px/app-pairing.svg [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M1.1 1C0.49248 1 0 1.49249 0 2.1V12.9C0 13.5075 0.492487 14 1.1 14H3C3.55228 14 4 13.5523 4 13C4 12.4477 3.55228 12 3 12H2V3H13V12H9C8.44772 12 8 12.4477 8 13C8 13.5523 8.44772 14 9 14H13.9C14.5075 14 15 13.5075 15 12.9V2.1C15 1.49248 14.5075 1 13.9 1H1.1Z" fill="#111418"/>
+<path d="M6.1 19C5.49248 19 5 18.5075 5 17.9V7.1C5 6.49249 5.49249 6 6.1 6H11C11.5523 6 12 6.44771 12 7C12 7.55229 11.5523 8 11 8H7V17H18V8H17C16.4477 8 16 7.55229 16 7C16 6.44771 16.4477 6 17 6H18.9C19.5075 6 20 6.49249 20 7.1V17.9C20 18.5075 19.5075 19 18.9 19H6.1Z" fill="#111418"/>
+</svg>
```
Comment (ggdouglas) [inline resources/icons/20px/app-pairing.svg]: Same here:
```svg
<svg viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg">
<path d="M1.1 1C0.49248 1 0 1.49249 0 2.1V12.9C0 13.5075 0.492487 14 1.1 14H3C3.55228 14 4 13.5523 4 13C4 12.4477 3.55228 12 3 12H2V3H13V12H9C8.44772 12 8 12.4477 8 13C8 13.5523 8.44772 14 9 14H13.9C14.5075 14 15 13.5075 15 12.9V2.1C15 1.49248 14.5075 1 13.9 1H1.1Z"/>
<path d="M6.1 19C5.49248 19 5 18.5075 5 17.9V7.1C5 6.49249 5.49249 6 6.1 6H11C11.5523 6 12 6.44771 12 7C12 7.55229 11.5523 8 11 8H7V17H18V8H17C16.4477 8 16 7.55229 16 7C16 6.44771 16.4477 6 17 6H18.9C19.5075 6 20 6.49249 20 7.1V17.9C20 18.5075 19.5075 19 18.9 19H6.1Z"/>
</svg>
```
app-pairing

File: resources/icons/16px/app-pairing.svg (hunk 1 above)
Comment (paja1t) [reply resources/icons/16px/app-pairing.svg]: done!

File: resources/icons/20px/app-pairing.svg (hunk 2 above)
Comment (paja1t) [reply resources/icons/20px/app-pairing.svg]: done!

**Review by ggdouglas — APPROVED** [review]

**Pull request merged**